ui/src/app/components/sidenav-view-assessment/sidenav-view-assessment.component.html

```diff
@@ -70,49 +64,249 @@ <h3 class="sidenav-title">Conversion status by table</h3>
+              class="danger">
+              error
+            </mat-icon>
+            ERRORS
+          </mat-panel-title>
+        </mat-expansion-panel-header>
+        <table mat-table [dataSource]="issueTableData_Errors" multiTemplateDataRows class="sidenav-databaseDefinitions">
 
+          <ng-container matColumnDef="position">
+            <th mat-header-cell *matHeaderCellDef class="mat-position"> No. </th>
+            <td mat-cell *matCellDef="let element" class="mat-position"> {{element.position}} </td>
+          </ng-container>
+
+          <ng-container matColumnDef="description">
+            <th mat-header-cell *matHeaderCellDef class="mat-description"> Description </th>
+            <td mat-cell *matCellDef="let element" class="mat-description"> {{element.description}} </td>
+          </ng-container>
+
+          <ng-container matColumnDef="tableCount">
+            <th mat-header-cell *matHeaderCellDef class="mat-tableCount"> Table Count </th>
+            <td mat-cell *matCellDef="let element" class="mat-tableCount"> {{element.tableCount}} </td>
+          </ng-container>
+
+          <ng-container matColumnDef="expand">
+            <th mat-header-cell *matHeaderCellDef aria-label="row actions">&nbsp;</th>
+            <td mat-cell *matCellDef="let element">
+              <button mat-icon-button aria-label="expand row" (click)="toggleRow(element); $event.stopPropagation()">
+                <mat-icon *ngIf="!isRowExpanded(element)">keyboard_arrow_down</mat-icon>
+                <mat-icon *ngIf="isRowExpanded(element)">keyboard_arrow_up</mat-icon>
+              </button>
+            </td>
+          </ng-container>
+
+          <ng-container matColumnDef="expandedDetail">
+            <td mat-cell *matCellDef="let element" [attr.colspan]="columnsToDisplayWithExpand.length">
+              <div class="example-element-detail" [@detailExpand]="isRowExpanded(element) ? 'expanded' : 'collapsed'">
+                <div class="example-element-description"> TABLES: {{element.tableNamesJoinedByComma}}</div>
+              </div>
+            </td>
+          </ng-container>
+
+          <tr mat-header-row *matHeaderRowDef="columnsToDisplayWithExpand"></tr>
+          <tr mat-row *matRowDef="let element; columns: columnsToDisplayWithExpand;" class="example-element-row"
+            [class.example-expanded-row]="isRowExpanded(element)" (click)="toggleRow(element)"></tr>
+          <tr mat-row *matRowDef="let row; columns: ['expandedDetail']" class="example-detail-row"></tr>
+        </table>
+      </mat-expansion-panel>
+      <br *ngIf="issueTableData_Errors.length">
+      <mat-expansion-panel *ngIf="issueTableData_Warnings.length">
+        <mat-expansion-panel-header>
+          <mat-panel-title>
+            <mat-icon matTooltip="Warning : Changes made because of differences in source and spanner capabilities."
```

Comment:
Let's reword it to: Warning : Schema modifications owing to difference in source and spanner capabilities.
Thoughts: @darshan-sj